File: MVVM/app/src/main/java/com/osos/mvvm/NileshViewModel.java
package com.osos.mvvm;

import android.database.Observable;

import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

import java.util.ArrayList;
import java.util.List;

public class NileshViewModel extends ViewModel {

   MutableLiveData<String> obj=new MutableLiveData<>();




    public MutableLiveData<String> getData(){

        obj.setValue("Nilesh Teji ");
        return  obj;



    }





    
}

File: KotlinApplication/app/src/main/java/com/osos/kotlinapplication/NileshViewModel.kt
package com.osos.kotlinapplication

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class NileshViewModel(a:String): ViewModel() {


   var data= MutableLiveData<String>();
    var a=a;

    fun getUser(): MutableLiveData<String> {
        data.value=a;
        return data;
    }

    fun setData(name:String){
        data.value=name;
    }



}